bot/exts/help_channels/_cog.py

```diff
@@ -275,6 +287,15 @@ async def init_cog(self) -> None:
         # This may confuse users. So would potentially long delays for the cog to become ready.
         self.close_command.enabled = True
 
+        # Getting channels that need to be included in the dynamic message.
+        task = asyncio.create_task(self.update_available_help_channels())
+        self.queue_tasks.append(task)
+
+        await task
+
+        log.trace("Dynamic available help message updated.")
+        self.queue_tasks.remove(task)
```

Comment:
What's the reason for creating a task and immediately awaiting it? If it's because of `queue_tasks` and their cancellation then it should be unnecessary, as the init task itself is cancelled when the cog is unloaded

Reply:
I thought back then that it would be necessary for this task to be cancelled if the cog was unloaded. 

I now see that there's no significant reason to do so.